src/components/Button.tsx

```diff
@@ -4,7 +4,7 @@ type ButtonProps = {
 }
 function Button({ placeholder, type = 'button', }: ButtonProps) {
     return (
-        <button type={type}>
+        <button className="border-2 text-white rounded-3xl my-8 px-8 p-2" type={type}>
             {placeholder}
         </button>
```

Comment:
What if you want to disable the button or the button on hero page, will it be possible to use this component